packages/client/src/arpa_reporter/views/ReportingPeriodsView.vue

```diff
@@ -97,7 +98,7 @@
           <div class="modal-footer">
             <button
               type="button"
-              class="btn btn-primary"
+              class="btn usdr-btn-primary"
```

Comment:
Note to reviewer: before/after screenshots

## Before
![image](https://github.com/user-attachments/assets/cbe295c9-d796-4970-b26f-906da05534ff)

Repeated for each row
![image](https://github.com/user-attachments/assets/9834022c-540d-4e7e-a723-1b12ef84935c)

![image](https://github.com/user-attachments/assets/f5a88a65-b565-4ffe-944a-5be73ee9a71f)


## After
![image](https://github.com/user-attachments/assets/e7674dd2-9308-46b0-b010-d4736325e784)


-------
Note:  This finding will be addressed in a separate PR

![image](https://github.com/user-attachments/assets/67cd4895-cdc0-4f6f-8fd8-626294487a4a)